local_claude_proxy.py

```
@@ -0,0 +1,74 @@
+
+@app.get("/health")
+async def health_check():
+    """Health check endpoint"""
+    try:
+        response = requests.get(f"{VAST_API_URL}/health", timeout=5)
+        return {
+            "status": "healthy",
+            "vast_backend": "connected" if response.status_code == 200 else "disconnected",
+            "timestamp": time.time()
+        }
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)